lib/Common/Memory/Recycler.cpp

```diff
@@ -7586,7 +7585,6 @@ Recycler::InitializeProfileAllocTracker()
         trackerDictionary = NoCheckHeapNew(TypeInfotoTrackerItemMap, &NoCheckHeapAllocator::Instance, 163);
 
 #pragma prefast(suppress:6031, "InitializeCriticalSectionAndSpinCount always succeed since Vista. No need to check return value");
-        InitializeCriticalSectionAndSpinCount(&trackerCriticalSection, 1000);
```

Comment:
>1000 [](start = 71, length = 4)

Not sure if this "1000" has any impact. If you support spinCount, you could use it here and not changing any behavior.